sklearn/cluster/_optics.py

```diff
@@ -131,42 +131,42 @@ class OPTICS(ClusterMixin, BaseEstimator):
         Note: fitting on sparse input will override the setting of
         this parameter, using brute force.
 
-    leaf_size : int, optional (default=30)
+    leaf_size : int, default=30
         Leaf size passed to :class:`BallTree` or :class:`KDTree`. This can
         affect the speed of the construction and query, as well as the memory
         required to store the tree. The optimal value depends on the
         nature of the problem.
 
-    n_jobs : int or None, optional (default=None)
+    n_jobs : int or None, default=None
         The number of parallel jobs to run for neighbors search.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
     Attributes
     ----------
-    labels_ : array, shape (n_samples,)
+    labels_ : array of shape (n_samples,)
```

Comment:
I forgot these ones (`ndarray` instead of `array`).

Reply:
ok